pkg/agent/datamodel/types.go

```diff
@@ -403,10 +493,10 @@ type MasterProfile struct {
 
 // OrchestratorProfile contains Orchestrator properties
 type OrchestratorProfile struct {
-	OrchestratorType    string                `json:"orchestratorType"`
-	OrchestratorVersion string                `json:"orchestratorVersion"`
-	KubernetesConfig    *api.KubernetesConfig `json:"kubernetesConfig,omitempty"`
-	DcosConfig          *api.DcosConfig       `json:"dcosConfig,omitempty"`
+	OrchestratorType    string            `json:"orchestratorType"`
+	OrchestratorVersion string            `json:"orchestratorVersion"`
+	KubernetesConfig    *KubernetesConfig `json:"kubernetesConfig,omitempty"`
+	DcosConfig          *api.DcosConfig   `json:"dcosConfig,omitempty"`
```

Comment:
we can remove this later

Reply:
Do you mean all 4 fields (OrchestratorType, OrchestratorVersion, KubernetesConfig, and DcosConfig) or just the DcosConfig field?